### ISSUE TITLE
Creating shops in others plots

### ISSUE THREAD

**Issue by clr1107** [EpicEricEE/ShopChest#273]
Players can create shops on others' plots without permission. I have flagged the plots create-shop OWNERS and use-shop EVERYONE and it's still not working. I also, even tried putting create-shop deny on the region (\_\_global\_\_) and it still did not stop them.

At this point, I'm thinking of disabling worldguard integration and just going off plotsquared; although this is definitely not ideal as players can create shops where I don't want them to. Edit: this didn't work either.

I am on 1.8.8 and therefore using WorldGuard 6.1 and ShopChest 1.12.3

**Comment by EpicEricEE** [comment]
Could you try if the issue still occurs on the latest snapshot builds? You can get it here: https://ci.codemc.org/job/EpicEricEE/job/ShopChest/lastSuccessfulBuild/

(If you plan on going back to v1.12.3, backup the database firstly and restore it when going back.)

**Comment by clr1107** [comment]
As my version of worldguard is old, I got this error starting up: https://hastebin.com/nenixuqaqu.apache
And no, it still allows them, despite the flags.

**Comment by clr1107** [comment]
This error was generated when they created the shop, perhaps due to the flags? 
https://hastebin.com/pizipoyuqe.apache

**Comment by EpicEricEE** [comment]
Your WorldGuard version does not support custom flags of ShopChest, so you may need to disable WorldGuard integration. AFAIK, you only need to set PlotSquared flags.

**Comment by clr1107** [comment]
Yes, however if I do this, players can create shops wherever else, e.g. in mines and that's not possible. Also, I tried that already. IIRC that error did not come up with the previous version, unless I never noticed it.

I think i've solved it. Players rank B and above are the ones this exploit works for. I was confused as to why it was now fixed for me but not somebody else.

They have these permissions extra:
plots.permpack.basic
plots.plot.1 
plots.use

Is it plots.use?
